compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmConstants.java

```diff
@@ -274,7 +275,7 @@
     public static final String RECORD_INIT_WRAPPER_NAME = "$init";
     public static final String LISTENER_REGISTRY_VARIABLE = "$listenerRegistry";
     public static final String CONFIGURE_INIT = "$configureInit";
-    public static final String CONFIGURATION_CLASS_NAME = "$ConfigurationMapper";
+    public static final String CONFIGURATION_CLASS_NAME = "$configurationMapper";
```

Comment:
This is a class name right? why are we changing it to lower case?

Reply:
For classes like `$_init` we use lowercase first letter. So as per the suggestion from @warunalakshitha  I changed it to the same convention.